src/apps/slackapp/slackapp/bolt_app.py

```diff
@@ -155,12 +160,25 @@ def event_test(client, say, event):
     # users with a daily limitation can be allowed to use in a different workspace
 
     if can_excute:
-        # used to reconstruct the question. if the question contains a link recreate
-        # them so that they contain scraped and summarized content of the link
-        reconstructor = PromptReconstructor(
-            question=question, slack_message=[replies["messages"][-1]]
-        )
-        question = reconstructor.reconstruct_prompt()
+        if "files" in event:
+            files = event['files']
+            if files[0]['size'] > cfg.FILE_SIZE_LIMIT:
+                say("Sorry, the file you attached is larger than 2mb. Please try again with a smaller file" , thread_ts=thread_ts)
+                return
+            file_prompt = QuestionWithFileHandler( question=question , team_id=team_id , user_id=user_id, files=files , token=cfg.SLACK_OAUTH_TOKEN )
+            file_prompt_data = file_prompt.reconstruct_prompt_with_file()
+            if file_prompt_data['status']=='success':
+                question =  file_prompt_data['data']
+            else:
+                say(file_prompt_data['message'] , thread_ts=thread_ts)
```

Comment:
If I understand correctly, this code reads the file and logs its content back to slack. If this is the case, it may be worthwhile to put it into another file and just call the function here.